jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -716,3 +717,26 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_ha(){
+    if [ ${JBPM_CLUSTER} = 'true' ]; then
```

Comment:
this env should be on the clear env function like the others.
plus, keep the same pattern for the other configuration envs

KIEGROUP_ENV_NAME

Reply:
ok